### ISSUE TITLE
Not able to create new custom effect...

### ISSUE THREAD

**Issue by Gacky2k** [DFreds/dfreds-convenient-effects#119]
**Describe the bug**
When I click the button at the top of the convenient effects window, to create a new effect.  Nothing happens...the smaller window is not opening to begin the process

**To Reproduce**
1. Click the open hand icon to open custom effects, the custom effects window will open correctly.  Next click the button at the top to "Create Effect".  The editor window is not displaying.

**Expected behavior**
The editor window should open to begin creating the custom effect.

**Screenshots**
Nothing to screenshot, except maybe the console log.  So instead, I have attached it.  It appears to throw an error with DAE, when you click the "New Effect" button.  The error is near the end of the attached log.

For what it's worth...when I disable DAE and reload, it works again!  But I need DAE for midi-qol, etc.

**Additional context**
[localhost-1643568745322.log](https://github.com/DFreds/dfreds-convenient-effects/files/7966708/localhost-1643568745322.log)

**Comment by DFreds** [comment]
That sounds like a DAE issue then

**Comment by Gacky2k** [comment]
Hmmm...I install and update from within Foundry.  Is it not updating to the newest?  To confirm that that wasn't the issue yesterday, before I posted...I uninstalled DAE then went into modules and searched DAE and reinstalled and it reinstalled v.0.8.82 yesterday.  And still had the issue.  After reading the replies today...I just went in to do it again and it had an update this time to v.0.8.83!!  I updated to this version and tested it out and it's WORKING!!!  Whatever update was done to DAE here the last day or two seems to have fixed the issue!!  

Thanks guys!!
